ioctl/util/util.go

```diff
@@ -64,6 +78,31 @@ func ConnectToEndpoint(secure bool) (*grpc.ClientConn, error) {
 	return grpc.Dial(endpoint, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})))
 }
 
+func GetAPIClientAndContext() (iotexapi.APIServiceClient, context.Context, error) {
```

Comment:
i think let's not complicate things too much for now, just create a new APIServiceClient and Context and return them, the purpose for now is to simplify this duplicate code in ioctl module

Reply:
Remove the gRPC keep-alive feature and initialize a new gRPC connection each time the function is called?